graylog2-server/src/main/java/org/graylog2/indexer/indices/jobs/SetIndexReadOnlyAndCalculateRangeJob.java

```diff
@@ -70,7 +65,7 @@ public void execute() {
             LOG.debug("Not running job for closed index <{}>", indexName);
             return;
         }
-        final SystemJob setIndexReadOnlyJob = setIndexReadOnlyJobFactory.create(indexName);
+        final SetIndexReadOnlyJob setIndexReadOnlyJob = setIndexReadOnlyJobFactory.create(indexName);
```

Comment:
The SetIndexReadOnlyJob actually calls the OptimizeIndexJob? :exploding_head: 

What a mess.
So we would need to call this `SetIndexReadOnlyThenOptionallyOptimizeAndCalculateRangeJob` :zany_face: